rclcpp/test/rclcpp/topic_statistics/test_subscription_topic_statistics.cpp

```diff
@@ -57,6 +57,9 @@ constexpr const std::chrono::seconds kTestDuration{10};
 constexpr const uint64_t kNumExpectedMessages{8};
 constexpr const uint64_t kNumExpectedMessageAgeMessages{kNumExpectedMessages / 2};
 constexpr const uint64_t kNumExpectedMessagePeriodMessages{kNumExpectedMessages / 2};
+constexpr const std::chrono::seconds kTransitionDuration{
```

Comment:
I think the naming of this constant is too specific to the context of https://github.com/ros2/rclcpp/issues/1531. Let's name it to something like `kUnstableMessageAgeWindowDuration`.